idea/api-gateway/src/config/configuration.ts

```diff
@@ -22,4 +22,8 @@ export default () => ({
       password: checkEnv('KAFKA_SASL_PASSWORD'),
     },
   },
+  // every 1 hour "0 * * * *"
+  cron: {
+    time: checkEnv('CRON_TIME') || '0 * * * *',
```

Comment:
In this case the default value will not work, because `checkEnv` will throw an error if variable isn't unspecified